<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optoins</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame11">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabelYear">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Year</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditYear">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabelMonth">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Mons, Day:mmdd</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMonth">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabelW">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>W</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditW">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabelIG">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>IG</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditIG">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxWIG">
        <property name="text">
         <string>InPut W,IG</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="frame5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxB0">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>B0 - Table option</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxNI">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Ni - Standard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxTE">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Te - Standard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxNE">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Ne - Standard Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxF1">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Standard F1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxD">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>D-region: IRI-95</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxStorm">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>F0F2 storm model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout">
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton5">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton5_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame12">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxES">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Es-Layer Foricast</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="buttonGroupF2">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>F2-Layer Foricast</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonU">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>URSI</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonC">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>CCIR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxWIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditYear</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxWIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditW</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxWIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditIG</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton5</sender>
   <signal>pressed()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton5_2</sender>
   <signal>pressed()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
